### ISSUE TITLE
Playback history and Open... dialogs should be merged

### ISSUE THREAD

**Issue by antonkudin** [iina/iina#2902]
When reopening IINA I often select streams or videos to play. IINA opens suggestion window with Open.. functions. To select past streams I have to select Playback History from the menu.

![Screen Shot 2020-04-27 at 18 43 54](https://user-images.githubusercontent.com/8632082/80392305-a52ffe00-88b7-11ea-9dca-f817502a06a4.png)

Both dialogs currently split these functions but they essentially do the same thing and should be merged, show past videos AND urls streams to be easily accessible right after launching.

**Comment by marcinretecki** [comment]
I agree and would like to add that from Playback history, you can not add items to current playlist. It is only possible to replace (play now).
